example/src/main/config/example-tenants.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "tenant": "DEFAULT_TENANT",
+    "data": {
+      "enabled": "true"
+    }
+  },
+  {
+    "tenant": "HTTP_TENANT",
+    "data": {
+      "enabled": "true"
+    },
+    "adapters": {
+      "http": {
+        "enabled": true,
+        "device-authentication-required": true
+      },
+      "mqtt": {
+        "enabled": false,
+        "device-authentication-required": true
+      },
+      "kura": {
```

Comment:
the type name of the Kura adapter is *hono-kura-mqtt* (see https://www.eclipse.org/hono/user-guide/kura-adapter/#downstream-meta-data)

Reply:
:+1: